src/vcpkg/base/system.process.cpp

```diff
@@ -635,6 +635,8 @@ namespace vcpkg
             "GameDKLatest",
             "GRDKLatest",
             "GXDKLatest",
+            "GameDKCoreLatest",
```

Comment:
`GameDKCoreLatest` is also associated with the new layouts and we should pass it through as this is used by the 'new layout' for the public version of the GDK.